packages/portal/src/pages/item/_.vue

```diff
@@ -356,27 +357,207 @@
       },
 
       async fetchMetadata() {
+        const params = {};
+        if (this.translatingMetadata) {
+          params.profile = 'translate';
+          params.lang = this.metadataLanguage;
+        }
+
+        let data;
         try {
-          const response = await this.$apis.record.getRecord(
-            this.identifier,
-            { locale: this.$i18n.locale, metadataLanguage: this.$auth.loggedIn ? this.$route.query.lang : undefined }
-          );
+          data = await this.$apis.record.get(this.identifier, params);
+        } catch (error) {
+          const errorResponse = error.response;
+          if (errorResponse?.status === 502 && errorResponse?.data?.code === '502-TS' && !this.fromTranslationError) {
+            this.fromTranslationError = true;
+            data = await this.$apis.record.get(this.identifier);
+          } else {
+            return this.$error(error, { scope: 'item' });
+          }
+        }
+        const edm = data.object;
+
+        if (this.identifier !== edm.about) {
+          return this.redirectToAltRoute({ params: { pathMatch: edm.about?.slice(1) } });
+        }
+
+        this.identifier = edm.about;
```

Comment:
Why is this needed? Just before this line there is a redirect in place for when the `identifier` and `edm.about` are different. Wouldn't that already resolve them to be the same?

Reply:
Indeed. That's removed.